src/ui/public/domLocation.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function DomLocationProvider($window) {
```

Comment:
$location doesn't provide the whole url, just the url as the angular router sees it. Since we have different apps with potentially different base urls (like status page) I needed the whole url